### PR TITLE
Update v0.1.9 - Final Jump Height Factor For Level 2 (Greece)

### PR DESCRIPTION
**Pull request by AkhilKulkarni123**
Had previously changed the jump height factor but wanted to make them nicer numbers.

### CODE REVIEW

File: assets/js/platformer/PlayerBase.js
Comment (Parallaxes) [inline assets/js/platformer/PlayerBase.js:1]: Haven't seen how much this affects the game, but I think it'll work. Will have to let other reviewers approve first.

**Review by Parallaxes — APPROVED** [review]
LGTM, will wait for other reviewers to approve the jump height change.

File: assets/js/platformer/PlayerBase.js
Comment (AkhilKulkarni123) [reply assets/js/platformer/PlayerBase.js:1]: Yes, minimal changes to lava speed and height change and sprite changes for now, nothing crazy. Just wanted to decrease the difficulty of the easy part of the game and will touch up on it after the next few changes on the level.

**Review by AdityaS-2010 — APPROVED** [review]
Looks fine, I think for the lava rise though it is too slow, maybe next time change it to 500

**Review by Shay-Robostrike1 — COMMENTED** [review]
this all looks good. some of the things could make more sense but it works, and we don't need perfection

**Review by Shay-Robostrike1 — APPROVED** [review]
looks good. some things are a bit odd but they work

**Pull request merged**